app/views/layouts/application.html.haml

```diff
@@ -0,0 +1,21 @@
+!!!
+%html
+  %head
+    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
+    %title InstagramChallenge
+    %link{:rel => 'stylesheet', :href => "https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css"}
+    = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true
+    = javascript_include_tag 'application', 'data-turbolinks-track' => true
+    = csrf_meta_tags
+  %body
+    %nav.navbar.navbar-default.navbar-fixed-top
+      - if user_signed_in?
+        = link_to 'Sign out', destroy_user_session_path, method: :delete
+      - else
+        = link_to 'Sign in', new_user_session_path
+        = link_to 'Sign up', new_user_registration_path
+    %div.container
+      = alert
+      = notice
+    %div.container#main.panel.panel-default
```

Comment:
Classes should be listed before IDs (.panel should precede #main)<br>`%div.container#main.panel.panel-default` can be written as `.container#main.panel.panel-default` since `%div` is implicit